src/Grid/grid.jl

```diff
@@ -300,6 +308,7 @@ struct Pyramid <: AbstractCell{RefPyramid}
     nodes::NTuple{5, Int}
 end
 
+geometric_interpolation(::Type{Point}) = Lagrange{RefPoint, 1}()
```

Comment:
Since it is a constant value, shouldn't this be 0th order?
```suggestion
geometric_interpolation(::Type{Point}) = Lagrange{RefPoint, 0}()
```